files/fielddata/script/scr_seq/scr_seq_0843_T20R0101.s

```diff
@@ -432,13 +432,19 @@ _05D9:
 	gender_msgbox msg_0543_T20R0101_00104, msg_0543_T20R0101_00105
 	goto _05F2
 
+// This script contains a programming mistake where if the player has
+// no space in their bag, the Everstone from Elm is permanently lost.
+// This is unlike every other script for obtaining an item from someone,
+// which all attempt to give the item until the player can accept it.
+// FLAG_GOT_EVERSTONE_FROM_ELM is set here but no code in the game ever
+// checks what it is, meaning that flag is useless.
```

Comment:
Be aware that comments like these may be deleted, as scripts are redumped occasionally to reflect changes in argument types.

Reply:
Got it, I'll leave those comments to the commit details.